helpers/tf_cfg.py

```diff
@@ -54,6 +54,8 @@ def defaults(self):
                                           'hostname': 'localhost',
                                           'ab': 'ab',
                                           'wrk': 'wrk',
+                                          'h2load': 'h2load',
+                                          'tls-perf': 'tls-perf',
```

Comment:
`tls-perf` isn't distributed in packages, so we need to specify the full path to the built binary. Surely you don't need the path and this line has no sense

Reply:
One can specify the path in `tests_config.ini`.
However the `check_dependencies.py` script ends with exception if `tls-perf` is not specified in `tf_cfg.py` at least in this shortened fake way.

Reply:
OK, I reviewed your trace
```
Traceback (most recent call last):
  File "./check_cmds_nodes.py", line 35, in <module>
    "cmd" : tf_cfg.cfg.get("Client", "h2load"),
  File "../helpers/tf_cfg.py", line 96, in get
    return self.config[section][opt]
  File "/usr/local/lib/python2.7/dist-packages/backports/configparser/__init__.py", line 1359, in __getitem__
    raise KeyError(key)
KeyError: 'h2load'
Status: some nodes have missing packages.
```
and it seems that `check_cmds_nodes.py` has `tls-perf`. So I agree that we should have `tls-perf` here. This config won't work, but the trace should go. Let's leave the fix as it is.